com.unity.render-pipelines.high-definition/Runtime/Lighting/SphericalHarmonics.cs

```diff
@@ -30,6 +30,60 @@ public static void GetCornetteShanksPhaseFunction(ZonalHarmonicsL2 zh, float ani
         }
     }
 
+    [Serializable]
+    [GenerateHLSL]
```

Comment:
this class should be move to Core package.

Reply:
I'm going to remove these SphericalHarmonicsL0, and SphericalHarmonicsL1 definitions. They turned out to not be needed at all, once I moved to storing SH coefficients as flat float arrays C# side, and as float4 coefficient arrays hlsl side.

Once removed, this class won't have any relation to this PR - Should I still move this class to Core in this PR as a general HDRP cleanup task?